src/js/config.js

```diff
@@ -0,0 +1,2 @@
+export const KEY = '98acf4edff3eebfa3ce173e73fc696cb';
+export const BASE_URL = 'http://api.openweathermap.org/data/2.5/forecast?';
```

Comment:
It's better to use secure `https` url otherwise it will throw a mixed content error when served from a secure domain.

Reply:
Thank you for the kind words and a great review. I will make the necessary changes as soon as I'm done with the final project.